src/App.js

```diff
@@ -24,7 +26,7 @@ import preloadSources from './action-creators/preloadSources'
   const src = urlDataSource()
   const localStateLoaded = src
     ? store.dispatch(loadFromUrl(src))
-    : loadLocalState()
+    : store.dispatch(loadLocalState())
```

Comment:
Since both cases call `store.dispatch` now, you can move it to the outside of the ternary.

Reply:
Sure